handlers/load_test_handler.go

```diff
@@ -195,9 +193,9 @@ func (h *Handler) LoadTestHandler(w http.ResponseWriter, req *http.Request, pref
 	}
 	loadTestOptions.Duration, err = time.ParseDuration(fmt.Sprintf("%d%s", tt, dur))
 	if err != nil {
-		logrus.Errorf("Error: unable to parse load test duration: %v", err)
-		http.Error(w, "unable to process the received data", http.StatusForbidden)
-		return
+		h.log.Error(ErrParseBool(err,obj))
```

Comment:
Consider the `obj := "load test duration"`  and  `h.log.Error(ErrParseBool(err, obj))`